.travis.install.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# If PHP 5.3.3 is installed, SSL/TLS isn't available to PHP.
+# Use a newer version of PHP for installing Composer dependencies.
+using_php_533="false"; [[ "$(php --version)" == PHP\ 5.3.3\ * ]] && using_php_533="true"
+if "$using_php_533"; then
+    echo "Using newer version of PHP for installing dependencies"
+    phpenv global 5.3
+    php --version
+
+    # Update Composer since the attempt made by the Travis setup script
+    # using PHP 5.3.3 would have failed.
+    #
+    # First, we update to the latest developer version since the version
+    # of Composer pre-installed on Travis systems (1.0.0) doesn't support
+    # selecting an update channel. Then, we use this version to rollback to
+    # the latest stable version.
+    composer self-update
+    composer self-update --stable
+fi
+
+# Install Composer dependencies.
+composer install
+
+# If using PHP 5.3.3 for testing purposes, stop using the newer PHP version.
+if "$using_php_533"; then
```

Comment:
do we support php 5.3.3 anymore? I though that the xdmod7.0 release was the last one.